lxd/device/unix_hotplug.go

```diff
@@ -230,27 +236,32 @@ func (d *unixHotplug) loadUnixDevice() *udev.Device {
 	}
 
 	devices, _ := e.Devices()
-	var device *udev.Device
+	var matchingDevices []*udev.Device
 	for i := range devices {
-		device = devices[i]
+		device := devices[i]
 
-		if device == nil {
+		if device == nil || device.Devnum().Major() == 0 || device.Devnode() == "" || strings.HasPrefix(device.Subsystem(), "usb") {
```

Comment:
Please can you add a comment here explaining the *why* for skipping in these cases.

Reply:
Sure thing!